src/lib/channels.py

```diff
@@ -83,9 +83,10 @@ class Channels(object):
     }
 
     @staticmethod
-    def timestamp_is_in_ranges(timestamp, ranges):
+    def timestamp_is_in_ranges(timestamp, ranges, matching_ranges):
         for range in ranges:
             if isoparse(range[0]) <= timestamp < isoparse(range[1]):
+                matching_ranges.append(range)
```

Comment:
It might be cleaner not to pass matching_ranges in, but instead to `return True, range` instead. I'm not sure, maybe check with Luke before changing.

Reply:
This was essentially for catching a list of matching time ranges.